src/components/media-types/container.js

```diff
@@ -37,8 +63,10 @@ export const Container = ({ children = null, interactive }) => {
 
   useEffect(() => {
     document.addEventListener('fullscreenchange', fullscreenChange, false)
+    document.addEventListener('webkitfullscreenchange', fullscreenChange, false)
     return () => {
       document.removeEventListener('fullscreenchange', fullscreenChange, false)
+      document.removeEventListener('webkitfullscreenchange', fullscreenChange, false)
```

Comment:
if webkit is not supported, why do we need this part?

Reply:
webkit on desktop and ipad does work, its just iphone. need to listen to this extra event to make it work properly